docs/legacy-account-migration.md

```diff
@@ -0,0 +1,25 @@
+# Legacy Account Migration
+
+## How to check an accounts current balance and other information
+
+Account balances migrated from Creditcoin 1.7 at block NNNN. Your starting balance on Creditcoin 2.0 should match your Creditcion 1.7 block NNNN balance
+
```

Comment:
IDK how to check balance on CC 1.7. Is this something that's documented or widely known by existing miners? Maybe worth a reference so they don't ask for help and can verify that balances have indeed been migrated properly.

Reply:
I have no idea how this is done. @nathanwhit any input?

Reply:
This is done through the old `ccclient`, the current [docs](https://docs.creditcoin.org/creditcoin-miners-manual#retrieve-your-miner-sighash-and-balance) have info on it. It should be (assuming you have a working node that's up to date and you've configured it with your private key) `./ccclient show Balance 0`. It should also be possible to look it up through the block explorer (if it's working)